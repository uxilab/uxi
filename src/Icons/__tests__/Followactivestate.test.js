import React from 'react';
import Followactivestate from '../Followactivestate';

describe('Icon : <Followactivestate />', () => {
  it('match snapshot', () => {
    expect(shallow(<Followactivestate />)).toMatchSnapshot();
  });
});

