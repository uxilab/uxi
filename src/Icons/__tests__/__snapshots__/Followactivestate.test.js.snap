// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Followactivestate /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0ZM8,14.6A6.6,6.6,0,1,1,14.6,8,6.61,6.61,0,0,1,8,14.6Z"
        />
        <path
          d="M9.62,4.17a2.24,2.24,0,0,0-1.47.62L8,4.94l-.15-.15a2.24,2.24,0,0,0-1.47-.62,2.86,2.86,0,0,0-3,3c0,2.68,3.9,5.1,4.34,5.37a.59.59,0,0,0,.6,0c.45-.27,4.34-2.69,4.34-5.37A2.86,2.86,0,0,0,9.62,4.17Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
